R/expressions.R

```diff
@@ -298,6 +302,16 @@ rollupResolution <- function (x) {
     }
 }
 
+#' @rdname expressions
+#' @export
+setMethod("rollupResolution<-",  "DatetimeVariable",
+    function(x, value = c('Y', 'M', 'D', 'h', 'm', 's', 'ms')) {
```

Comment:
This was literally just merged while you are out (so no wonder it's not here!), but we have a function that will automatically validate rollups. https://github.com/Crunch-io/rcrunch/blob/master/R/misc.R#L229-L237

I know that in general a list like this and then `match.arg()` is the right pattern, in this case we aren't getting much more than using the validator that exists. Specifically, the man pages for `*<-` methods don't include those values as far as I can see.